gene_expression_scatter_plot/gene_scatter_plot_home.html

```diff
@@ -13,6 +13,8 @@
 <script src="js/tToP.js"></script> 
 <script src="js/tableExport.js"></script>
 <script src="js/jquery.base64.js"></script>
+<!-- 刘莉莉添加epvalue.js -->
+<script src="js/epvalue.js"></script> 
```

Comment:
@BaekLi 

Where is `epvalue.js` ?

Please make a screenshot or a video to show that empirical p-value calculation works.

Hui